### DISCUSSION TITLE
How can I use my custom trained Stanford NER CRF model?

### DISCUSSION THREAD

**Discussion by venti07** [explosion/spaCy#10868] [Help: Coding & Implementations]
I have trained my own Stanford NER CRF model and would like to use it with spacy? Is this possible with spacy? 
I already found spacy-stanfordnlp but was not able to add my own model. With NLTK this is possible with StanfordNERTagger for example.

Thanks in advance for your help.

**Comment by adrianeboyd** [comment]
I haven't tried this directly, but you should be able to pass all the same arguments to `Pipeline` with `spacy_stanza.load_pipeline` as described here: https://github.com/explosion/spacy-stanza#stanza-pipeline-options

So however you load the custom pipeline in `stanza` should look very similar to how it can be loaded with `spacy_stanza` (probably a custom `dir` and `package`)?

**Reply by venti07** [reply]
The model is a `model.ser.gz` - file. Unfortunately I can't insert it into the pipeline. Therefore I am looking for a way to load this model with spacy.

**Reply by adrianeboyd** [reply]
Sorry, I misunderstood what kind of model it is. I don't think there's an existing wrapper to load this model in a spacy pipeline component.

If you're interested in developing a custom pipeline component, it looks like the current recommendation is to use the CoreNLP server and there are a number of existing wrappers/packages/demos for python: https://stanfordnlp.github.io/CoreNLP/other-languages.html#python
